src/styles/pages/_object-detail.scss

```diff
@@ -182,6 +182,10 @@ $metadata-collapsed-width: 36vw;
 
 	&__related {
 		max-width: 100vw;
+
+		.ScrollbarsCustom-TrackY > .ScrollbarsCustom-Thumb {
```

Comment:
You're already targetting the scrollbar here; avoid inline styles in the component